agreement_legal/models/agreement.py

```diff
@@ -350,7 +350,7 @@ def create_new_agreement(self):
             "stage_id": self.env.ref("agreement_legal.agreement_stage_new").id,
         }
         res = self.copy(default=default_vals)
-        res.sections_ids.clauses_ids.write({'agreement_id': res.id})
+        res.sections_ids.write({'agreement_id': res.id})
```

Comment:
```suggestion
        res.sections_ids.mapped('clauses_ids').write({'agreement_id': res.id})
```
I got bug form this line 
Step:
1.Create 2 Sections : Sections A and Sections B.
2.Create 3 Clauses to Sections B
3.Create New Agreement.
and I got this
![BUY-VOIP013  SIP-ISDN gateways - Odoo - Google Chrome_003](https://user-images.githubusercontent.com/52144935/64104529-959afc80-cd9e-11e9-9cef-561db436feec.png)

Reply:
@kongrattapong Thanks for reviewing. I'll check it ASAP

Reply:
@kongrattapong thanks for catching & solving this bug. If everything looks fine please approve.